kubeflow/jupyter/tests/jupyter_test.py

```diff
@@ -112,19 +112,21 @@ def test_jupyter(env, namespace):
   host = api_client.configuration.host
   logging.info("Kubernetes master: %s", host)
   master = host.rsplit("/", 1)[-1]
+  service = "jupyter-test"
 
-  this_dir = os.path.dirname(__file__)
-  app_dir = os.path.join(this_dir, "test_app")
-
-  ks_cmd = ks_util.get_ksonnet_cmd(app_dir)
+  with open("jupyter_test.yaml") as params: 
+    wf_result = yaml.load(params)
+    namespace = wf_result['metadata']['namespace']
+    name = wf_result['metadata']['name']
+    k8s_apps_v1 = k8s_client.AppsV1Api()
+    resp = k8s_apps_v1.create_namespaced_deployment(
+      body=wf_result, namespace=namespace)
+    logging.info("Deployment created: status='%s'" % resp.metadata.name)
 
-  name = "jupyter-test"
-  service = "jupyter-test"
-  component = "jupyter"
-  params = ""
-  ks_util.setup_ks_app(app_dir, env, namespace, component, params)
+  # this_dir = os.path.dirname(__file__)
+  # app_dir = os.path.join(this_dir, "test_app")
 
-  util.run([ks_cmd, "apply", env, "-c", component], cwd=app_dir)
```

Comment:
Can we delete the old jsonnet file?

Reply:
Yes, I removed `jupyter_test.jsonnet`, `sync-notebook_test.jsonnet`, and `notebooks_test.jsonnet`. Let me know if you meant just the `jupyter_test.jsonnet`.